File: problems/merge-sorted-arrays/boilerplate-full/function.js

  ##USER_CODE_HERE##
  
  const input = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n').join(' ').split(' ');
  const arr1 = parseInt(input.shift());
  const arr2 = parseInt(input.shift());
  const result = mergeSortedArrays(arr1, arr2);
  console.log(result);
      

File: problems/merge-sorted-arrays/boilerplate-full/function.rs

  use std::io::{self, BufRead};
  
  ##USER_CODE_HERE##
  
  fn main() {
    let stdin = io::stdin();
    let mut input = stdin.lock().lines().map(|l| l.unwrap());
    let arr1: unknown = input.next().unwrap().parse().unwrap();
  let arr2: unknown = input.next().unwrap().parse().unwrap();
    let result = mergeSortedArrays(arr1, arr2);
    println!("{}", result);
  }
      